.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "Java",
+
+	"image": "mcr.microsoft.com/devcontainers/java:0-17",
```

Comment:
Is `0-17` a range? It should be no less than 17 for us

Reply:
No, 0 is related the version of the container itself, the JDK version is 17 only. (verified with java --version command)